spring-boot-ecommerce/src/main/java/com/luv2code/ecommerce/dao/ProductRepository.java

```diff
@@ -14,4 +14,5 @@ public interface ProductRepository extends JpaRepository<Product, Long> {
 
     Page<Product> findByNameContaining(@RequestParam("name") String name, Pageable pageable);
```

Comment:
No need to create method use already existing method like findById inside JpaRepository.

Read all mehtod inside JpaRepository and how jpa works